lib/index.js

```diff
@@ -19,6 +19,7 @@ var defaults = {
   debug: identity,
   lint: true,
   minify: false,
+  encapsulateStyles: true,
```

Comment:
I am changing my mind a bit, maybe for v3 we should make this feature opt-in and experimental i.e. `false` by default and if it works well enable it by default in v4?